### PR TITLE
Adding condition to rawScores to display on tooltip

### PR DESCRIPTION
**Pull request by Emily-ejag**
## Proposed changes
If the rawScore === 0 then it will be displayed on the tooltip

## Types of changes

What types of changes does this pull request introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Documentation Update
- [ ] Tests (new or updated tests)
- [ ] Style (changes to code styling)
- [ ] CI (continuous integration changes)
- [ ] Repository Maintenance
- [ ] Other (please describe below)

## Checklist

- [x] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
- [x] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
- [x] Linting checks pass with my changes.
- [x] Any existing unit tests pass with my changes.
- [x] Any existing end-to-end tests pass with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
- [ ] I have added JSDoc comments as appropriate.
- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
- [x] I have shared this PR on the roar-pr-reviews channel (if I have access)
- [ ] I have linked relevant issues (if any)

### CODE REVIEW

File: src/components/reports/TableScoreTag.vue [hunk 1]
```diff
@@ -78,6 +78,7 @@ function handleToolTip(_taskId, _toolTip, _colData) {
   // if score exists
   if (
     _colData.scores?.[_taskId]?.rawScore ||
+    _colData.scores?.[_taskId]?.rawScore === 0 ||
```
Comment (lucasxsong) [inline src/components/reports/TableScoreTag.vue:81]: If I am understanding this correctly, could we reduce an extra conditional if we just replace the first conditional with if `...?.rawScore != undefined`?

Comment (Emily-ejag) [reply src/components/reports/TableScoreTag.vue:81]: yes, good catch!

Comment (Emily-ejag) [inline src/components/reports/TableScoreTag.vue:81]:
```suggestion
    _colData.scores?.[_taskId]?.rawScore != undefined ||
```

File: src/components/reports/TableScoreTag.vue [hunk 2]
```diff
@@ -78,6 +78,7 @@ function handleToolTip(_taskId, _toolTip, _colData) {
   // if score exists
   if (
     _colData.scores?.[_taskId]?.rawScore ||
```
Comment (Emily-ejag) [inline src/components/reports/TableScoreTag.vue:80]:
```suggestion

```

**Comment by lucasxsong** [comment]
On this PR's staging link, it now appears that the Phoneme scores are not showing up. Not sure if this was introduced by this PR or not, but I will investigate on staging. 

Screenshot 2024-06-25 at 9 29 43 AM

**Review by lucasxsong — APPROVED** [review]

**Pull request merged**